src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -153,10 +152,21 @@ object DatabaseConversions extends LazyLogging {
 
         val untouchedAccounts =
           inactiveBakers
-            .map(_.copy(isActiveBaker = Some(false)))
-            .filterNot(accountsHistoryRow => touched.contains(accountsHistoryRow.accountId))
+            .filterNot(inactiveAccountsRow => touched.contains(inactiveAccountsRow.accountId))
```

Comment:
style: I guess we could simply use `row` as identifier, without the ceremony, since it's a local and it's very clear from the context what it is.

Reply:
ok, changed